nav_msgs/Odometry pose
float64 tf_height
